frontera/contrib/scrapy/schedulers/__init__.py

```diff
@@ -0,0 +1,30 @@
+from scrapy.core.scheduler import Scheduler
+from scrapy.utils.misc import load_object
+
+
+DOWNLOADER_MIDDLEWARE = 'frontera.contrib.scrapy.middlewares.schedulers.SchedulerDownloaderMiddleware'
+SPIDER_MIDDLEWARE = 'frontera.contrib.scrapy.middlewares.schedulers.SchedulerSpiderMiddleware'
+
+
+class BaseFronteraScheduler(Scheduler):
```

Comment:
I like the intention, probably you wanted to simplify Frontera setup. Forced addition of downloader/spider middlewares makes the whole middleware stack behavior unpredictable and counter-intuitive. Keep in mind, that our SchedulerSpiderMiddleware, affects behavior of spider middlewares processing: it consumes all links, and bypasses only items. So potentially it could break the behavior in a unexpected way, especially for middlewares after inserted one.